__tests__/app.test.js

```diff
@@ -373,4 +373,51 @@ describe('/api/articles/:articleid/comments', () => {
 
     });
 
+});
+
+describe('/api/comments/:comment_id', () => {
+
+    describe('DELETE comment', () => {
+        test('/api/comments/:comment_id responds 204 and delete comment ', () => {
+            return request(app).delete('/api/comments/1').expect(204).then((response) => {
```

Comment:
Nice! You'd be welcome to check the database directly from the tests if you'd like to check it's definitely gone